.stylelintrc.json

```
@@ -0,0 +1,28 @@
+{
+    "rules": {
+        "string-quotes": "double",
+        "selector-pseudo-element-no-unknown": true,
+        "declaration-block-trailing-semicolon": "always",
+        "no-invalid-double-slash-comments": true,
+        "property-no-vendor-prefix": true,
+        "at-rule-no-vendor-prefix": true,
+        "value-no-vendor-prefix": true,
+        "media-feature-name-no-vendor-prefix": true,
+        "order/order": [
+            [
+                "custom-properties",
+                "declarations"
+            ]
+        ],
+        "rule-empty-line-before": ["always", {"except": ["after-single-line-comment", "inside-block-and-after-rule", "first-nested"], "ignore": ["inside-block"]}],
```

```suggestion
        "rule-empty-line-before": [
            "always",
            {
                "except": [
                    "after-single-line-comment",
                    "inside-block-and-after-rule",
                    "first-nested"
                ],
                "ignore": [
                    "inside-block"
                ]
            }
        ],
```
Can we unflatten this line just to be consistent :)